<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistBuilder</class>
 <widget class="QWidget" name="PlaylistBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="8">
    <widget class="QPushButton" name="btnAddFolder">
     <property name="text">
      <string>Import  Folder</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="11">
    <widget class="QFrame" name="framePlaylistbuilder">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="11">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="tblSongsTable">
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ARTIST</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TITLE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DURATION</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tblPlaylistSongs">
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ARTIST</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>TITLE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DURATION</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item row="3" column="9">
    <widget class="QPushButton" name="btnSavePlaylist">
     <property name="text">
      <string>Save Playlist</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="btnAddFile">
     <property name="text">
      <string>Add File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="11">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>8</height>
      </size>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnPrevious">
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnPlaySample">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btnRemove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4" colspan="2">
    <widget class="QLabel" name="lblInfo">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
